### PR TITLE
Allow specification of helm position.

### PR DESCRIPTION
**Pull request by ralesi**
Provide variable to define preferred position for helm to appear.

### CODE REVIEW

File: core/templates/.spacemacs.template [hunk 1]
```diff
@@ -110,6 +110,9 @@ before layers configuration."
    dotspacemacs-use-ido nil
    ;; If non nil the paste micro-state is enabled. When enabled pressing `p`
    ;; several times cycle between the kill ring content.
+   dotspacemacs-helm-position 'bottom
+   ;; define the position to display `helm', options are `bottom', `top',
+   ;; `left', or `right'.
```
Comment (mbertheau) [inline core/templates/.spacemacs.template]: comment should be on top

File: spacemacs/packages.el [hunk 2]
```diff
@@ -1584,6 +1584,7 @@ Removes the automatic guessing of the initial value based on thing at point. "
       (defvar spacemacs-helm-display-buffer-regexp `("*.*helm.**"
                                                      (display-buffer-in-side-window)
                                                      (inhibit-same-window . t)
+                                                     (side . ,dotspacemacs-helm-position)
```
Comment (bmag) [inline spacemacs/packages.el]: You may want to add a `window-width` parameter as well

**Comment by ralesi** [comment]
Yes,  good catch,  and also this will  interfere with the autoresize option if placed on the side.  Will submit a patch tonight.

**Comment by syl20bnr** [comment]
Thank you again, we have now a pretty set of dotspacemacs variables to quickly personalize the helm feeling, I love them :+1: 
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by ralesi** [comment]
@syl20bnr we have one unmerged commit here.  Can you merge the second one?

**Comment by ralesi** [comment]
Nevermind, I will submit another PR for it.

**Comment by ralesi** [comment]
Here #2913

**Pull request closed without merging**